metabase/metabase
# Polish timeline events in collections

Epic: https://github.com/metabase/metabase/issues/20289

Changes:
- Adds min-height to timeline list and timeline details modals
- Adds scrolling to timeline list and timeline details modals
- Changes modal header icons
- Prevents modals from closing by clicking on Undo
- Fixes routing when submitting a modal form

How to test:
- Go to a collection, e.g. to the root collection
- Click on the events (calendar) icon
- Add a timeline with an event
- When the event list is displayed, make sure it has some min height and the footer (e.g. dinosaur) is at the bottom
- Add ~10 events. Make sure the list is scrolled correctly
- Try archiving an event by using the menu near the item. Make sure it's possible to click on `Undo` and the modal won't close.
- Try archiving several events, then use the header menu to view all archived events. Delete several events (confirm deletion). Press the browser's back button. Make sure you are navigated to the timeline, and not the delete modals

<img width="660" alt="Screenshot 2022-02-28 at 16 51 33" src="https://user-images.githubusercontent.com/8542534/155994720-031edb78-7a37-44f7-be09-9df93cbaffc4.png">
